src/xenia/cpu/compiler/passes/constant_propagation_pass.cc

```diff
@@ -648,6 +648,14 @@ bool ConstantPropagationPass::Run(HIRBuilder* builder, bool& result) {
             result = true;
           }
           break;
+        case OPCODE_AND_NOT:
+          if (i->src1.value->IsConstant() && i->src2.value->IsConstant()) {
+            v->set_from(i->src1.value);
+            v->AndNot(i->src2.value);
```

Comment:
To keep the `Value` class simpler and to avoid introducing a somewhat possibly unsupported (although I'm not yet sure, but `Value` contains more fields than what `set_from` copies, and I'm not sure if you can just duplicate them) concept of constructing a `Value` from another `Value`, what do you think about implementing the "and not" logic here instead?
```cpp
v->set_from(i->src2.value);
v->Not();
v->And(i->src1.value);
```

Reply:
Thanks! Yeah. It seems reasonable for it to be there. It's only used here and it is short and easy to understand anyway. Changed & Removed dedicated AndNot